ir/instr.cpp

```diff
@@ -2860,7 +2769,10 @@ void Load::print(std::ostream &os) const {
 }
 
 StateValue Load::toSMT(State &s) const {
-  auto &p = s.getAndAddPoisonUB(*ptr).value;
+  // NOTE: We neeed clarification whether loading a partially undefined pointer
+  // is UB. Memory sanitizer has an option (detecting a partially undef pointer)
+  // for this, so worth discussing it.
+  auto &p = s.getAndAddPoisonUB(*ptr, true).value;
```

Comment:
I would say it's weird to store to an undef pointer, even if just 1 bit of the offset is undef.
Maybe submit a patch to LangRef to clarify this so we are all on the same page.

Reply:
Okay. I have a noundef patch that I'm writing (in addition to the malloc), I'll submit a patch to LangRef after that